libraries/chain/include/eosio/chain/peer_keys_db.hpp

```diff
@@ -16,28 +16,37 @@ class peer_keys_db_t {
    using peer_key_map_t = boost::unordered_flat_map<name, peer_info_t, std::hash<name>>;
    using new_peers_t    = flat_set<name>;
```

Comment:
Can we make it a `boost::unordered_flat_set`?

Reply:
Actually no. We use `std::set_union` which doesn't work for `unordered_*`.

Reply:
Where do we use `new_peers_t ` for `std::set_union`?

Reply:
I did go ahead and rename to `peer_name_set_t` and use throughout.

https://github.com/AntelopeIO/spring/blob/596135dc238ffb98c07dc31fab353011b536500e/plugins/net_plugin/include/eosio/net_plugin/auto_bp_peering.hpp#L422-L422

Reply:
https://github.com/AntelopeIO/spring/pull/1265/commits/596135dc238ffb98c07dc31fab353011b536500e